io.github.dida_code.OpstaKultura.json

```diff
@@ -0,0 +1,38 @@
+{
+    "app-id" : "io.github.dida_code.OpstaKultura",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "opstakultura",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "opstakultura",
```

Comment:
Project license is missing from appdata, it won't cause an error but you should include it.

Also, please pull the screenshots from a tag/commit instead of `main`